local-dev/secretstores.bicep

```diff
@@ -47,5 +47,6 @@ output result object = {
     type: daprType
     version: daprVersion
     metadata: daprComponent.spec.metadata
+    componentName: daprComponent.metadata.name
```

Comment:
Is this required? And if so should we add it to all the other dapr recipes?

Reply:
its not required, removed it